prime-dotnet-webapi/Services/EnrolleeService.cs

```diff
@@ -411,21 +411,11 @@ public async Task<int> GetEnrolleeCountAsync()
                 .CountAsync();
         }
 
-        public async Task<Enrollee> UpdateEnrolleeAdjudicator(int enrolleeId, Admin admin = null)
+        public async Task<Enrollee> UpdateEnrolleeAdjudicator(int enrolleeId, Nullable<int> adminId = null)
```

Comment:
```int?``` is shorthand for Nullable<int>

Reply:
and updated similar method in SiteService